test/example.spec.ts.snapshot

```diff
@@ -23,10 +23,9 @@ xquery version "3.1";
 module namespace body = "http://e-editiones.org/roaster/body";
 import module namespace errors = "http://e-editiones.org/roaster/errors";
 
-declare
-  variable $bla := "bla"; (: A variable to check if it works :)
+declare variable $bla := "bla"; (: A variable to check if it works :)
 
-(:~
+  (:~
```

Comment:
I think the initial two whitespaces are incorrect, according to the xqdoc specs. https://xqdoc.org/xquery-style.pdf p. 11

Reply:
yeah, that is still broken

Reply:
The newline after the block is also wrong

Reply:
But the overall result is _less_ broken

Reply:
That document you are referencing is non-normative IMHO

Reply:
Comments in general should be aligned as well: if the formatter chooses to change the indent level, the content should be aligned with the first option.

Reply:
I believe the main issue with the comments having that trailing newline is that they are attached to the Separator before the item. While they should be attached to the following AnnotatedDecl.

Just a hypothesis